rllib/execution/train_ops.py

```diff
@@ -65,6 +65,104 @@ def train_one_step(trainer, train_batch) -> Dict:
     return info
 
 
+@ExperimentalAPI
+def multi_gpu_train_one_step(trainer, train_batch) -> Dict:
+    config = trainer.config
+    workers = trainer.workers
+    local_worker = workers.local_worker()
+    policies = local_worker.policies_to_train
+    num_sgd_iter = config.get("sgd_num_iter", 1)
+    sgd_minibatch_size = config.get("sgd_minibatch_size", 0)
+
+    # Collect actual GPU devices to use.
+    if not config.get("num_gpus", 0):
+        _fake_gpus = True
+        num_gpus = 1
+    type_ = "cpu" if _fake_gpus else "gpu"
+    devices = [
+        "/{}:{}".format(type_, 0 if _fake_gpus else i)
```

Comment:
can we have multiple fake gpus?

Reply:
yes, `_fake_gpus` also works for a multi-gpu (>1) setup.

Reply:
Actually, we don't even need the devices themselves, just the number of 'em.
Will simplify.

Reply:
`_fake_gpus` here meant basically: no GPUs (which is a misleading var name).
This is all simplified now and the `_fake_gpu` var is no longer needed.